src/Sylius/Bundle/AdminBundle/Resources/views/promotion/form.html.twig

```diff
@@ -0,0 +1,23 @@
+{# Rendered with using \Sylius\Bundle\AdminBundle\TwigComponent\Promotion\PromotionType #}
```

Comment:
```suggestion
{# Rendered with using \Sylius\Bundle\AdminBundle\TwigComponent\Promotion\FormComponent #}
```

Reply:
I'll use `Renederd with` or `Rendered using` because `Rendered with using` does not sound right